buf.gen.yaml

```diff
@@ -0,0 +1,17 @@
+version: v2
+managed:
+  enabled: false
+plugins:
+  # Custom comment extraction plugin - ONLY generate comments, not .pb.go files
+  # All .pb.go files come from buf.build/gen/go/redpandadata/core/protocolbuffers/go
+  - local: tools/go-pbgen/run-plugin.sh
```

Comment:
The protos are being generated from the local proto files or from the ones in buf?

Reply:
That would be great, I'll investigate how to do it and possibly send something today. Thanks!